test/AddressRegistry.Tests/BackOffice/Handlers/WhenChangingAddressPosition.cs

```diff
@@ -0,0 +1,102 @@
+namespace AddressRegistry.Tests.BackOffice.Handlers
+{
+    using System;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AddressRegistry.Api.BackOffice.Abstractions;
+    using AddressRegistry.Api.BackOffice.Abstractions.Requests;
+    using AddressRegistry.Api.BackOffice.Handlers;
+    using Autofac;
+    using AutoFixture;
+    using Be.Vlaanderen.Basisregisters.CommandHandling;
+    using Be.Vlaanderen.Basisregisters.CommandHandling.Idempotency;
+    using Be.Vlaanderen.Basisregisters.GrAr.Edit.Contracts;
+    using FluentAssertions;
+    using global::AutoFixture;
+    using Infrastructure;
+    using SqlStreamStore;
+    using SqlStreamStore.Streams;
+    using StreetName;
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class WhenChangingAddressPosition : AddressRegistryBackOfficeTest
+    {
+        private readonly TestBackOfficeContext _backOfficeContext;
+        private readonly IdempotencyContext _idempotencyContext;
+        private readonly TestMunicipalityConsumerContext _municipalityConsumerContext;
+        private readonly IStreetNames _streetNames;
+
+        public WhenChangingAddressPosition(ITestOutputHelper testOutputHelper) : base(testOutputHelper)
+        {
+            Fixture.Customize(new WithFixedMunicipalityId());
+
+            _idempotencyContext = new FakeIdempotencyContextFactory().CreateDbContext();
+            _backOfficeContext = new FakeBackOfficeContextFactory().CreateDbContext();
+            _municipalityConsumerContext = Container.Resolve<TestMunicipalityConsumerContext>();
+            _streetNames = Container.Resolve<IStreetNames>();
+        }
+
+        [Fact]
+        public async Task GivenRequest_ThenPersistentLocalIdETagResponse()
+        {
+            var streetNamePersistentLocalId = new StreetNamePersistentLocalId(123);
+            var addressPersistentLocalId = new AddressPersistentLocalId(456);
+            var niscode = new NisCode("12345");
+            var postalCode = new PostalCode("2018");
+            var houseNumber = new HouseNumber("11");
+
+            _backOfficeContext.AddressPersistentIdStreetNamePersistentIds.Add(
+                new AddressPersistentIdStreetNamePersistentId(addressPersistentLocalId, streetNamePersistentLocalId));
+            _backOfficeContext.SaveChanges();
+
+            var municipalityId = Fixture.Create<MunicipalityId>();
+            var gml =
+                "<gml:Polygon srsName=\"https://www.opengis.net/def/crs/EPSG/0/31370\" xmlns:gml=\"http://www.opengis.net/gml/3.2\">" +
```

Comment:
Maybe create a `GeometryHelper` class where you can add this like `ValidGmlPoint` or something.
Seems to be repeated a few times.

Reply:
Added `ValidGmlPolygon `to `GeometryHelpers`.